ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host2.cfg

```diff
@@ -66,18 +71,112 @@ dot1x
    radius av-pair service-type
    radius av-pair framed-mtu 1500
 !
+interface Port-Channel4
+   no switchport
+   ip address 10.9.2.3/31
+   isis enable EVPN_UNDERLAY
+   isis circuit-type level-2
+   isis metric 50
+   isis network point-to-point
+!
+interface Port-Channel5
+   no switchport
+   ip address 10.9.2.5/31
+   isis enable EVPN_UNDERLAY
+   isis metric 50
+   isis passive
+!
+interface Port-Channel6
+   no switchport
+   ip address 10.9.2.7/31
+   isis enable EVPN_UNDERLAY
+   isis circuit-type level-1-2
+   isis metric 100
+!
 interface Dps1
    description Test DPS Interface
    no shutdown
    mtu 666
    flow tracker sampled FT-S
    ip address 192.168.42.42/24
 !
+interface Ethernet1
+   description P2P_LINK_TO_EAPI-SPINE1_Ethernet1
+   mtu 1500
+   no switchport
+   ip address 172.31.255.1/31
+   isis enable EVPN_UNDERLAY
+   isis metric 50
+   isis network point-to-point
+!
+interface Ethernet2
+   description P2P_LINK_TO_EAPI-SPINE2_Ethernet1
+   mtu 1500
+   no switchport
+   ip address 172.31.255.3/31
+   isis enable EVPN_UNDERLAY
+   isis circuit-type level-1-2
+   isis metric 50
+   isis network point-to-point
+!
+interface Ethernet3
+   description MLAG_PEER_EAPI-LEAF1B_Ethernet3
+   channel-group 3 mode active
+!
+interface Ethernet4
+   channel-group 4 mode active
+!
+interface Ethernet5
+   channel-group 5 mode active
+!
+interface Ethernet6
+   channel-group 6 mode active
```

Comment:
Can you check the use of these interfaces? it has only channel-group mode

Reply:
Remove as all keys are covered already.